### ISSUE TITLE
Reserved keywords as properties of module.exports are transpiled into named exports, which causes "unexpected token" errors. 

### ISSUE THREAD

**Issue by lxe** [rollup/rollup-plugin-commonjs#116]
Consider:

``` javascript
module.exports.delete = 'foo';
```

This will get transpiled into:

``` javascript
var delete = 'foo';

var mycjsmodule = {
    delete: delete
};

export default mycjsmodule;
export { mycjsmodule as __moduleExports };
export { delete };
```

Which will cause a SyntaxErrors on `var delete = 'foo';` and `export { delete };`

~~Since it's impossible to do `import { delete } from './mycjsmodule'` anyways, it would be safe to not create named exports for reserved-keyword properties of module.exports.~~ EDIT: Apparently it's possible to import a reserved keyword like that, as long as it's destructured, at least according to babel! Not sure if that's desired behavior.

**Comment by lxe** [comment]
According to babel, this is valid es2015:

```javascript
import { this } from 'wat'

const foo = 'wat';
export { lol as const };
```

This is madness!

**Comment by Rich-Harris** [comment]
> This is madness!

Seconded! In any case, this should be fixed now in 6.0.0. Thanks
